tests/Cache/CacheMemcachedStoreTest.php

```diff
@@ -74,6 +74,20 @@ public function testSetMethodProperlyCallsMemcache()
         Carbon::setTestNow();
     }
 
+    public function testPutMethodReturnsFalseIfTTLIsNull()
+    {
+        if (! class_exists(Memcached::class)) {
```

Comment:
This isn't needed: the mock above seems to be already setting the expected methods